nep-x1.mediawiki

```diff
@@ -0,0 +1,137 @@
+<pre>
+  NEP: Undefined
+  Title: NFT Royalty Standard
+  Author: Owen Zhang<zhangzhihong@ngd.neo.org>
+  Type: Standard
+  Status: Draft
+  Created: 2022-09-05
+</pre>
+
+==Abstract==
+
+This NEP defines a global standard to get royalty payment information for non-fungible tokens (NFTs) to enable support for royalty payments across all NFT marketplaces in the NEO Smart Economy.
+
+==Motivation==
+
+Need support royalty information which is not only for royalty payments across NFT marketplaces, but also good to protect creative copyright. Currently, no standard for creators to declare loyalty to all marketplaces in their NFTs. They have to discuss the loyalty with different marketplaces, otherwise these marketplaces which haven't touched with the creators can't get royalty details and have to define the loyalty in their marketplace contract by themselves. This standard is compatible with [https://github.com/neo-project/proposals/blob/master/nep-11.mediawiki NEP-11]. 
+
+==Specification==
+
+===Common methods===
+
+====isGlobalRoyalty====
+
+<pre>
+{
+  "name": "isGlobalRoyalty",
+  "safe": true,
+  "parameters": [],
+  "returntype": "Boolean"
+}
+</pre>
+
+This function shows if this contract has <code>global royalty</code> or <code>specific royalty</code> to different tokenId. 
+
+If return <code>true</code>, then it has global royalty.  All tokenId MUST implement the same static royalty amount.
+
+If return <code>false</code>, then it has specific royalty. MUST store specific royalty amount for each tokenId, static or dynamic, either is OK .
+
+====royaltyInfo====
+
+If <code>isGlobalRoyalty</code> return true, this function should be:
+
+<pre>
+{
+  "name": "royaltyInfo",
+  "safe": true,
+  "parameters": [],
+  "returntype": "Array"
+}
+</pre>
+
+If <code>isGlobalRoyalty</code> return false, this function should be:
+
+<pre>
+{
+  "name": "royaltyInfo",
+  "safe": true,
+  "parameters": [
+    {
+      "name": "tokenId",
+      "type": "ByteString"
+    },
+    {
+      "name": "salePrice",
+      "type": "Integer"
+    }
+  ],
+  "returntype": "Array"
+}
+</pre>
+
+Returns a JSON Array which includes <code>royaltyRecipient</code> and <code>royaltyAmount</code>.
```

Comment:
Why is it a `JSON` if it's clearly a NeoVM array stack item with two elements inside?

Reply:
Done.